.github/workflows/masterbot.yml

```diff
@@ -37,7 +37,8 @@ jobs:
         run: |
           npm run nx:graph
           npm run release:phase1
-          npm run release:phase2
+          npm run release:phase3
```

Comment:
My understanding, looking at the changes (I'm leaving this comment mainly for my own benefit 😅):

- Old phase1 was:
  1. `publish:bump`
  2. `build`
  3. `publish:npm`

- Old phase2 was: `reify`

-----

- New phase1 is:
  1. `publish:bump`
  2. `build`

- New phase3 is `publish: npm`
- New phase4 is `reify`

-----

Therefore this appears to be entirely equivalent, which is what we want.